go.mod

```diff
@@ -272,4 +272,4 @@ require (
 
 replace github.com/tendermint/tendermint => github.com/bnb-chain/tendermint v0.31.12
 
-replace github.com/VictoriaMetrics/metrics => github.com/greyireland/metrics v0.0.5
+replace github.com/VictoriaMetrics/metrics => /Users/ashukla/polygon/fork/metrics
```

Comment:
Probably need remove?

Reply:
Yup, I'm waiting for this [PR](https://github.com/greyireland/metrics/pull/1) to get merged then I'll change it and merge after addressing all other comments